third-party/qthread/Makefile

```diff
@@ -193,35 +160,28 @@ clobber: FORCE
 
 
 qthread-config: FORCE
-	@test -z "$(FAILBUILD)" || ( echo 'compileline failed!' && false )
-
-#
-# These first few lines touch a bunch of autoconf-oriented files in a
-# certain order to prevent autoconf from running again; otherwise, we
-# ran into issues if a user's autoconf environment was not as far
-# ahead in version numbers as that which was used when packaging the
-# Qthreads release
-#
-	cd $(QTHREAD_SUBDIR) && touch -c configure.ac
-	cd $(QTHREAD_SUBDIR) && find . -name "*.m4" | xargs touch
-	sleep 1
-	cd $(QTHREAD_SUBDIR) && touch -c aclocal.m4
-	sleep 1
-	cd $(QTHREAD_SUBDIR) && touch configure
-	cd $(QTHREAD_SUBDIR) && find . -name "*.in" | xargs touch
-
-#
-# Then configure
-#
 	mkdir -p $(QTHREAD_BUILD_DIR)
-	cd $(QTHREAD_BUILD_DIR) \
-	&& $(QTHREAD_SUBDIR)/configure CC='$(CC)' CFLAGS='$(CFLAGS)' \
-	       CXX='$(CXX)'  CXXFLAGS='$(CFLAGS)' LDFLAGS='$(RUNTIME_LFLAGS) $(LDFLAGS)' \
-	       --prefix=$(QTHREAD_INSTALL_DIR) $(CHPL_QTHREAD_CFG_OPTIONS)
+	$(CMAKE) -B $(QTHREAD_BUILD_DIR) -S $(QTHREAD_SUBDIR) \
+	$(CHPL_QTHREAD_CFG_OPTIONS) \
+	-DQTHREADS_BUILD_TESTS=OFF \
+	-DCMAKE_INSTALL_PREFIX='$(QTHREAD_INSTALL_DIR)' \
+	-DCMAKE_INSTALL_LIBDIR='$(QTHREAD_LIB_DIR)' \
+	-DBUILD_SHARED_LIBS=OFF \
+	-DCMAKE_C_COMPILER='$(CC)' \
+	-DCMAKE_C_FLAGS='$(CFLAGS)' \
+	-DCMAKE_CXX_COMPILER='$(CXX)' \
+	-DCMAKE_CXX_FLAGS='$(CXXFLAGS)' \
+	-DCMAKE_MAKE_PROGRAM='$(MAKE)'
 
 qthread-build: FORCE
-	cd $(QTHREAD_BUILD_DIR) && $(MAKE)
-	cd $(QTHREAD_BUILD_DIR) && $(MAKE) install
+	if [ -f $(QTHREAD_BUILD_DIR)/Makefile ]; then cd $(QTHREAD_BUILD_DIR) && make; else $(CMAKE) --build $(QTHREAD_BUILD_DIR); fi
+
+qthread-install: FORCE
+	if [ -f $(QTHREAD_BUILD_DIR)/Makefile ]; then cd $(QTHREAD_BUILD_DIR) && make install; else $(CMAKE) --install $(QTHREAD_BUILD_DIR); fi
+
```

Comment:
Sorry I missed this on the last round, but this should use `$(MAKE)` instead of just `make`

Reply:
Thanks, will fix.